src/sonic-yang-models/yang-models/sonic-syslog.yang

```diff
@@ -0,0 +1,33 @@
+module sonic-syslog {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-system-syslog";
+    prefix syslog;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    container sonic-syslog {
+
+        container SYSLOG_SERVER {
+
+            description "SYSLOG SERVER part of config_db.json";
+
+            list SYSLOG_SERVER_LIST {
+                max-elements 10;
```

Comment:
Where does 10 come from?

Reply:
Remove the max-elements

Reply:
address in latest commit

Reply:
Also remove in the description ", limiting to 10" ?

Reply:
removed the description in the latest commit